> Log for le_failure
This is printed by test le_failure
> Test complete: le_failure exit code 1: FAILURE
---
